project/settings/production.py

```
@@ -0,0 +1,20 @@
+"""Production settings. Requires the 'prod' extra (gunicorn, whitenoise)."""
+
+import os
+
+from .base import *
```

Ruff (0.15.6)

[error] `from .base import *` used; unable to detect undefined names

(F403)